src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -151,11 +152,10 @@ export class CourseManagementRequests {
      * add text exercise to an exercise group in exam or to a course
      * @returns <Chainable> request response
      * */
-    createAndAddTextExerciseToExam(group: any, title = 'Text exercise ' + generateUUID()) {
-        const textExercise: any = { ...textExerciseTemplate, exerciseGroup: group };
-        textExercise.exerciseGroup = group;
-        textExercise.title = title;
-        return cy.request({ method: POST, url: BASE_API + 'text-exercises', body: textExercise });
+    createTextExercise(body: { course: any } | { exerciseGroup: any }, title = 'Text exercise ' + generateUUID()) {
+        const template: any = { ...textExerciseTemplate, title };
+        const templateWithBody = Object.assign({}, template, body);
```

Comment:
```suggestion
        const templateWithBody = this.getCourseOrExamExercise(template, body);
```

Reply:
Actually I got rid of the `getCourseOrExamExercise` method in [this PR](https://github.com/ls1intum/Artemis/pull/3951). The method only contains one line of code. Having it in a separate method does not provide any value in my opinion